src/ethereum/info/abstractInfo.ts

```diff
@@ -0,0 +1,120 @@
+import { EdgeCurrencyInfo, EdgeTokenMap } from 'edge-core-js/types'
+
+import { makeOuterPlugin } from '../../common/innerPlugin'
+import { createEvmTokenId, makeMetaTokens } from '../../common/tokenHelpers'
+import type { EthereumTools } from '../EthereumTools'
+import {
+  asEthereumInfoPayload,
+  EthereumFees,
+  EthereumInfoPayload,
+  EthereumNetworkInfo
+} from '../ethereumTypes'
+import {
+  evmCustomFeeTemplate,
+  evmCustomTokenTemplate,
+  evmMemoOptions
+} from './ethereumCommonInfo'
+
+const builtinTokens: EdgeTokenMap = {}
+
+// Fees are in Wei
+const networkFees: EthereumFees = {
+  default: {
+    baseFee: undefined,
+    baseFeeMultiplier: {
+      lowFee: '1',
+      standardFeeLow: '1.25',
+      standardFeeHigh: '1.5',
+      highFee: '1.75'
+    },
+    gasLimit: undefined, // Limits must always be estimated by eth_estimateGas
+    gasPrice: {
+      lowFee: '1000000001',
+      standardFeeLow: '40000000001',
+      standardFeeHigh: '300000000001',
+      standardFeeLowAmount: '100000000000000000',
+      standardFeeHighAmount: '10000000000000000000',
+      highFee: '40000000001',
+      minGasPrice: '10000000'
+    },
+    minPriorityFee: '2000000000'
+  }
+}
+
+const networkInfo: EthereumNetworkInfo = {
+  addressQueryLookbackBlocks: 60, // ~2 minutes
+  networkAdapterConfigs: [
+    {
+      type: 'rpc',
+      servers: ['https://api.mainnet.abs.xyz	']
+    },
+    {
+      type: 'evmscan',
+      servers: ['https://api.abscan.org/']
+    }
+  ],
+  uriNetworks: ['abstract'],
+  ercTokenStandard: 'ERC20',
+  chainParams: {
+    chainId: 2741,
+    name: 'abstract'
+  },
+  hdPathCoinType: 60,
+  amberDataBlockchainId: '',
+  pluginMnemonicKeyName: 'abstractMnemonic',
+  pluginRegularKeyName: 'abstractKey',
+  ethGasStationUrl: null,
+  networkFees
+}
+
+const currencyInfo: EdgeCurrencyInfo = {
+  canReplaceByFee: true,
+  currencyCode: 'ETH',
+  customFeeTemplate: evmCustomFeeTemplate,
+  customTokenTemplate: evmCustomTokenTemplate,
+  chainDisplayName: 'abstract',
+  assetDisplayName: 'Ethereum',
+  memoOptions: evmMemoOptions,
+  pluginId: 'abstract',
+  walletType: 'wallet:abstract',
+
+  // Explorers:
+  addressExplorer: 'https://abscan.org/address/%s',
+  transactionExplorer: 'https://abscan.org/tx/%s',
+
+  denominations: [
+    {
+      name: 'ETH',
+      multiplier: '1000000000000000000',
+      symbol: 'Ξ'
+    }
+  ],
+
+  // Deprecated:
+  defaultSettings: {
+    customFeeSettings: ['gasLimit', 'gasPrice'],
+    otherSettings: {
+      chainParams: networkInfo.chainParams,
+      ercTokenStandard: networkInfo.ercTokenStandard
+      // Skip networkAdapterConfigs
```

Comment:
Why skip networkAdapterConfigs? This will permanently break a bunch of staking plugins like Kiln, who are expecting to find servers in here.